src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -107,12 +104,29 @@ public UserDefault getUserDefaultSettings(final String userId) {
             LOG.error("Failed to retrieve user profile", e);
             final List<String> headerMessages = e.getResponseHeaders().get("Message");
             final String message = headerMessages != null ? headerMessages.get(0) : e.getMessage();
-            if (message != null)
+            if (message != null) {
                 throw new BadRequestException(message);
+            }
             throw new ServiceException("Problem getting user default settings for " + userId);
         }
     }
 
+    @Override
+    public SecurityClassification getHighestUserClassification() {
```

Comment:
minor, safe to ignore. 
I would have in repo methods only logic to deal with db access. A method like this, which adds further elaboration, would be better in the UserService. But I've seen we are doing this in other methods in this class already